eng/pipelines/runtime-community.yml

```diff
@@ -0,0 +1,59 @@
+trigger: none
```

Comment:
This template doesn't make sense to port it as it doesn't even have triggers for release/* branches.